libs/ngrx-toolkit/src/lib/with-call-state.ts

```diff
@@ -14,27 +14,27 @@ export type NamedCallStateSlice<Collection extends string> = {
 };
 
 export type CallStateSlice = {
-  callState: CallState
```

Comment:
Can you maybe find a way of not reformatting those files? We might have to come up with a separate PR which runs prettier with a default configuration over all files but maybe this one is not suited for it.

There are so many changes and I have a hard time to differentiate reformatting and actual modifications.

Reply:
I've reverted the changes and deactivated prettier for the workspace for now. I haven't thought about my autoformat on save setting after applying the rename for the `Empty` type.